src/test/java/VolodymyrHrytsiuk/Tests/CrowellUkraineCrisis.java

```diff
@@ -0,0 +1,28 @@
+package VolodymyrHrytsiuk.Tests;
+
+import BasePageAndTestInit.TestInit;
+import VolodymyrHrytsiuk.Tests.Pages.HomePageCrowell;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class CrowellUkraineCrisis extends TestInit {
+
+    @Test
+    public void checkUkraineCrisis() {
+        HomePageCrowell homePageCrw = new HomePageCrowell(driver);
+
+        openUrl("https://www.crowell.com/");
+        homePageCrw.ukrCrisisCenter().click();
+        driver.getWindowHandles().forEach(tab -> driver.switchTo().window(tab));
```

Comment:
Please move all driver.navigate().back() to TestInit. Create a custom method something like:
Public void navigateBack(){driver.navigate().back();} and use it instead
same for all driver methods